docker/.env

```diff
@@ -0,0 +1,11 @@
+# Docker container environment variables
+# The file is used for the docker-compose.yml::env_file setting
```

Comment:
BTW, why `.env` and not just `env` or `docker-compose.env`? Why hidden?

Reply:
Haha, just a consequence of following the example at docker.com blindly.